feel/.ruby-version

```diff
@@ -0,0 +1 @@
+3.4.2
```

Comment:
Might as well move this to the root with some minor path modifications so we don't have to repeat it.